packages/manager/modules/pci/src/projects/project/storages/object-storage/object-storage.routing.js

```diff
@@ -21,7 +24,11 @@ export default /* @ngInject */ ($stateProvider) => {
         ),
     resolve: {
       archive: () => false,
-
+      tagPageParams: /* @ngInject */ (trackPage, $transition$) => {
+        if ($transition$.params().trackingTag) {
+          trackPage($transition$.params().trackingTag);
+        }
+      },
```

Comment:
i guess we have more sense to complete this into `Ctrl.onInit()` function.
resolve your `tagPageParam` here and send it into your controller.